<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginDialog</class>
 <widget class="QDialog" name="loginDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QFrame" name="logoFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>260</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>30</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="accountLabel">
           <property name="text">
            <string>账号:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="accountText">
           <property name="inputMethodHints">
            <set>Qt::InputMethodHint::ImhNone</set>
           </property>
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Normal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="passwordLabel">
           <property name="text">
            <string>密码:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordText">
           <property name="maxLength">
            <number>32</number>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="messageLabel">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="margin">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>取消</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>loginDialog</receiver>
   <slot>onLoginButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>loginDialog</receiver>
   <slot>onCancelButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLoginButtonClicked()</slot>
  <slot>onCancelButtonClicked()</slot>
 </slots>
</ui>
